lib/routes/account.js

```diff
@@ -1263,6 +1263,7 @@ module.exports = function (
         db.account(uid)
           .then(
             function (account) {
+              var isEmailVerification = butil.buffersAreEqual(code, account.emailCode)
```

Comment:
Something about `isEmailVerification` name seems a little misleading to me. Would `isAccountVerficiation` make more sense?